tests/foreman/api/test_host.py

```diff
@@ -1387,6 +1387,31 @@ def test_positive_read_puppet_ca_proxy_name(
     assert session_puppet_enabled_proxy.name == host['puppet_ca_proxy_name']
 
 
+@pytest.mark.tier2
+def test_positive_list_hosts_thin_all(module_target_sat):
```

Comment:
Are you sure about the severity of this tests inherited from module docstrings, which is `high`?

I mean the bug itself was `medium`.